### DISCUSSION TITLE
propane usage calculations

### DISCUSSION THREAD

**Discussion by DavePanetta** [jgyates/genmon#1094] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).

### Version

Version V1.19.02

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have not submitted logs

### What is/are your question(s)?

I have 2 120 Gal propane tanks. My supplier will only fill them to 80%.
Does the usage calculation take this into account?
Should I use a 240 Gal Tank size or de rate the size to 192 Gal. - Just wondering.

**Comment by jgyates** [comment, marked as answer]
No, genmon only calculates based on the numbers you enter for the tank size and the reading on the sensor (if you are using a tank monitor). If you are using estimated fuel usage it only uses the tank size and does not factor in for limits of the tank for fuel type.

It is up to you to change the tank size if you chose to show an 80% filled tank as 100%. Just keep in mind that if for some reason your supplier fills the tank to 83%, genmon will not show 103%, it will assume the additional 3% is an error and round it down to 100%.

**Comment by DavePanetta** [comment]
Understanding is important.
Maybe I should look into setting up tank monitors.

Thank you for the clarification.
Dave

**Comment by jgyates** [comment]
FYI, there is also a setting on the Advanced Settings page (double click the wrench icon in the upper right) called "Subtract Fuel" that will subtract a specified amount of fuel from the tank before the calculation. You could set this to 48 (2x 20% of 120) after your tanks a filled. Of course a sensor would be more accurate.

**Comment by DavePanetta** [comment]
Thanks for the tip!
